control-plane-operator/controllers/hostedcontrolplane/oauth/deployment.go

```diff
@@ -3,6 +3,7 @@ package oauth
 import (
 	"context"
 	"fmt"
+	"github.com/openshift/hypershift/support/globalconfig"
```

Comment:
Can we keep a distinct import group for stdlib imports?

Reply:
done